config/routes.rb

```diff
@@ -20,6 +20,6 @@
 
   # match '/users/finish_signup' => 'users/registrations#finish_signup', via: [:get, :patch], :as => :finish_signup
 
-  ActiveAdmin.routes(self)
+  ActiveAdmin.routes(self) rescue ActiveAdmin::DatabaseHitDuringLoad
```

Comment:
Fix error `ActiveAdmin::DatabaseHitDuringLoad` when run "rake db:create"

Reply:
can you tell more about the case leads to this error?

Reply:
when calling: `ActiveAdmin.routes(self)` it then opens `app/admin/#{model}.rb` which attempts to connect to the DB and raises an error if DB don't exist yet.
<img width="721" alt="ảnh" src="https://user-images.githubusercontent.com/10648521/58191744-6f6ab880-7ce9-11e9-8421-533dc9c917ac.png">